lib/harmonium/table.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Harmonium.Table do
+  @moduledoc false
+
+  import Phoenix.HTML.Tag
+
+  def table(opts \\ [], do: block) do
+    table_element(:table, [class: "rev-Table"], opts, do: block)
+  end
+
+  def table_head(opts \\ [], do: block) do
+    table_element(:thead, [class: "rev-Table-head"], opts, do: block)
+  end
+
+  def table_body(opts \\ [], do: block) do
+    table_element(:tbody, [class: "rev-Table-body"], opts, do: block)
+  end
+
+  def table_row(opts \\ [], do: block) do
+    table_element(:tr, [class: "rev-Table-row"], opts, do: block)
+  end
+
+  def table_header(opts \\ [], do: block) do
+    table_element(:th, [class: "rev-Table-header"], opts, do: block)
+  end
+
+  def table_data(opts \\ [], do: block) do
+    table_element(:td, [class: "rev-Table-Data"], opts, do: block)
+  end
+
+  defp table_element(tag, default_opts, opts, do: block) do
+    opts = Keyword.merge(default_opts, opts)
+
+    content_tag tag, opts do
+      block
+    end
+  end
+end
```

Comment:
I should have said this before, but I think all these helper functions should follow the current standard Harmonium pattern where you can optionally pass CSS class modifiers as the 2nd arg:

```elixir
  @doc """
  Constructs a callout class.
  """
  def callout_class(modifiers \\ []), do: rev_class(@callout_class, modifiers)

  @doc """
  Renders a callout.
  """
  def callout(do: block), do: callout([], do: block)

  def callout(modifiers, do: block), do: callout(:div, modifiers, do: block)

  def callout(tag, modifiers, do: block) do
    content_tag tag, class: callout_class(modifiers) do
      block
    end
  end
```

Our Harmonium SCSS Table does include some very useful SUIT modifiers: https://github.com/revelrylabs/harmonium/blob/master/scss/components/_Table.scss